### ISSUE TITLE
emerge progress in title

### ISSUE THREAD

**Issue by c4pp4** [shiznix/unity-gentoo#145]
When I'm using emerge in graphical terminal (e.g. gnome-terminal, uxterm),
following snippet of code from _unity-gentoo/profiles/xenial/base/profile.bashrc_

`KEYWORD_STATE="$(emerge --info | grep ACCEPT_KEYWORDS)"`

is causing terminal window title shows **/usr/lib64/python2.7/site-packages**
instead of **emerge: (5 of 27) name-of/package-version Compile**.

Maybe it would suffice to change it to

`KEYWORD_STATE="$(grep "ACCEPT_KEYWORDS" /etc/portage/make.conf)"`

What do you think about it?

**Comment by shiznix** [comment]
Thank you!
Was seeing that also but had not had a chance to debug what was causing it.

'emerge --info' method was only used as old Gentoo installations had make.conf reside at /etc/make.conf instead of now /etc/portage/make.conf (both locations are still supported by portage)

But I guess enough time has passed that most installations are now using /etc/portage/make.conf ?

**Comment by c4pp4** [comment]
You are right with that locations. According to man settings from /etc/portage/make.conf will override settings from /etc/make.conf, so the code would be something like this
```
if ! KEYWORD_STATE="$(grep "ACCEPT_KEYWORDS" /etc/portage/make.conf)"; then
        KEYWORD_STATE="$(grep "ACCEPT_KEYWORDS" /etc/make.conf)"
fi
```

**Comment by shiznix** [comment]
Looks good will commit something shortly...

**Comment by shiznix** [comment]
Should be fixed in commit a408ec7952f57ab4debc974e3bf0dc5c7ef2b25d

Big thanks for troubleshooting and providing a fix :)
